.swift-version

```diff
@@ -0,0 +1 @@
+6.0.3
```

Comment:
This file is missing a trailing newline. Additionally, could it have a comment on the first or second line pointing to relevant documentation for discoverability?

Reply:
This file should be dead simple, able to be `cat`'ed as part of scripts/commands to easily get the desired toolchain without any further processing.